File: Chalkable.StiConnector/SyncModel/GradingScale.cs
namespace Chalkable.StiConnector.SyncModel
{
    public class GradingScale : SyncModel
    {
        public int GradingScaleID { get; set; }
        public int SchoolID { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public int? HomeGradeToDisplay { get; set; }

        public override int DefaultOrder => 21;
    }
}

File: Chalkable.Web/app/chlk/models/grading/GradingScale.js
REQUIRE('chlk.models.id.StandardsGradingScaleId');

NAMESPACE('chlk.models.grading', function () {
    "use strict";

    /** @class chlk.models.grading.GradingScale*/
    CLASS('GradingScale', [
        chlk.models.id.StandardsGradingScaleId, 'id',
        String, 'name',
        String, 'description',
        [ria.serialize.SerializeProperty('homegradetodisplay')],
        Number, 'homeGradeToDisplay'
    ]);
});
